model/lifecycle_test.go

```diff
@@ -1038,71 +934,6 @@ func TestCreateBuildFromVersion(t *testing.T) {
 
 		})
 
-		Convey("the build should contain task caches that correspond exactly"+
-			" to the tasks created", func() {
-
-			args := BuildCreateArgs{
-				Project:       *project,
-				Version:       *v,
-				TaskIDs:       table,
-				BuildName:     buildVar2.Name,
-				ActivateBuild: false,
-				TaskNames:     []string{},
-			}
-			build, tasks, err := CreateBuildFromVersionNoInsert(args)
-			So(err, ShouldBeNil)
-			So(build.Id, ShouldNotEqual, "")
-			So(len(tasks), ShouldEqual, 4)
-			So(len(build.Tasks), ShouldEqual, 4)
-
-			// make sure the task caches are correct.  they should also appear
-			// in the same order that they appear in the project file
-			So(build.Tasks[0].Id, ShouldNotEqual, "")
-			So(build.Tasks[0].DisplayName, ShouldEqual, "taskA")
-			So(build.Tasks[0].Status, ShouldEqual, evergreen.TaskUndispatched)
-			So(build.Tasks[1].Id, ShouldNotEqual, "")
-			So(build.Tasks[1].DisplayName, ShouldEqual, "taskB")
-			So(build.Tasks[1].Status, ShouldEqual, evergreen.TaskUndispatched)
-			So(build.Tasks[2].Id, ShouldNotEqual, "")
-			So(build.Tasks[2].DisplayName, ShouldEqual, "taskC")
-			So(build.Tasks[2].Status, ShouldEqual, evergreen.TaskUndispatched)
-			So(build.Tasks[3].Id, ShouldNotEqual, "")
-			So(build.Tasks[3].DisplayName, ShouldEqual, "taskE")
-			So(build.Tasks[3].Status, ShouldEqual, evergreen.TaskUndispatched)
-		})
-
-		Convey("a task cache should not contain execution tasks that are part of a display task", func() {
```

Comment:
assuming this condition is still true, we should also keep this test (although probably checking ID specifically now)

Reply:
I reinstated the test